plasmapy/examples/langmuir_analysis.py

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
+"""
+Analyzing Langmuir
+=========================
+
+Let's analyze various Langmuir probe characteristics using the
+`diagnostics.langmuir` subpackage.
+"""
+
+import astropy.units as u
+from plasmapy.diagnostics.langmuir import Characteristic, swept_probe_analysis
+import pickle
+import numpy as np
+
+######################################################
+"""The first characteristic we analyze is a simple single-probe measurement in
+a low (ion) temperature, low density plasma with a cylindrical probe. This
+allows us to utilize OML theory implemented in `swept_probe_analysis`.
+The data has been preprocessed with some smoothing, which allows us to obtain
+a nice Electron Energy Distribution Function (EEDF) as well.
+"""
+
+# Load the bias and current values stored in the .p pickle file.
+[bias, current] = pickle.load(open('langmuir_samples\\Beckers2017.p', 'rb'))
```

Comment:
I'm kinda wondering about the best location for these. `plasmapy/examples/langmuir_samples` seems weird.

Oh, and... do you develop on Windows? Because this path is probably not going to work on Linux, so we should change this to
```python
import os
path = os.path.join("langmuir_samples", "Beckers2017.p")
```
to be OS-independent.